dev-python/sphinxcontrib-serializinghtml/sphinxcontrib_serializinghtml-2.0.0.recipe

```diff
@@ -5,9 +5,9 @@ HOMEPAGE="http://sphinx-doc.org/
 	https://pypi.org/project/sphinxcontrib-serializinghtml/"
 COPYRIGHT="2007-2019 by the Sphinx team"
 LICENSE="BSD (2-clause)"
-REVISION="2"
-SOURCE_URI="https://pypi.python.org/packages/source/s/sphinxcontrib-serializinghtml/sphinxcontrib-serializinghtml-$portVersion.tar.gz"
-CHECKSUM_SHA256="aa5f6de5dfdf809ef505c4895e51ef5c9eac17d0f287933eb49ec495280b6952"
+REVISION="1"
+SOURCE_URI="https://github.com/sphinx-doc/sphinxcontrib-serializinghtml/archive/refs/tags/$portVersion.tar.gz"
```

Comment:
What is the reason for changing the URL to GitHub?

Reply:
Easy fix, prior URI also didn't work so I din't persue the check on pypi :) Could be changed.